app/scss/_eventsDetails.scss

```diff
@@ -1,294 +1,182 @@
-            padding-right: 50px;
-
-
-            p{
-              margin-top: 10px;
-              margin-bottom: 30px;
-              font-size: 20px;
-            }
-            h3{
-              margin-top: 60px;
-            }
-            ul{
-              list-style-image: url('../../img/check.resized.jpg');
-              li{
-                margin-bottom: 15px;
-              }
-            }
-          }
-
-
-
-
-
-
-
-
-
-        }
-
-      }
-
-      .event-sidebar {
-        display: inline-block;
-        width: 250px;
-        text-align: left;
-
-        .img-container{
-          position: relative;
-          width: 250px;
-          height: 250px;
-          display: inline-block;
-          margin-bottom: 10px;
-          img {
-            position: absolute;
-            width: 100%;
-            height: auto;
-          }
-          .border{
-            position: absolute;
-            width: 125px;
-            height: 125px;
-            bottom: -8px;
-            right: -8px;
-            background-color: blue;
-          }
-        }
-        div{
-          display: inline-block;
-          p{
-            font-size: 14px;
-          }
-          .name{
-            font-weight: bolder;
-          }
-
-          .description>p{
-            color: gray;
-            font-size: 14px;
-          }
-        }
+      margin-top: 15px;
+      margin-left: 25%;
+      margin-right: 25%;
+      border-bottom: solid $orange 3px;
+      box-sizing: border-box;
+    }
+  }
 
-        @media screen and (max-width: 1200px)
-        {
-          width: 900px;
-          margin-top: 100px;
-          margin-left: auto;
-          margin-right: auto;
-          .img-container {
-            margin-left: 50px;
-          }
+  &--description{
+    width: 65%;
+    margin: 0;
+    box-sizing: border-box;
+    text-align: left;
+  }
+  &--title{
+    font-weight: 800;
+    font-size: 20px;
+    margin-right: 50px;
+  }
+  &--author{
+    color: gray;
+    margin-top: 10px;
+    margin-bottom: 20px;
+  }
+  &--toggle{
+    color: gray;
+    text-decoration: underline;
 
-          .description-container{
-            display: inline-block;
-            width: 300px;
-            margin-left: 40px;
-          }
-        }
+  }
+  &--toggle:hover{
+    cursor: pointer;
+  }
+  &--more{
+    position: relative;
+    text-align: justify;
+    display: flex;
+    height: 0;
+    overflow: hidden;
+    color: gray;
+    font-size: 16px;
+    strong{
+      color: black;
+    }
+  }
 
+  &--about{
+    padding: 10px;
+    width: 50%;
+  }
+  &--learn{
+    padding: 10px;
+    width: 50%;
+    text-align: left;
+    ul{
+      list-style-type: none;
+    }
+    li{
+      position: relative;
+      margin-bottom: 10px;
+    }
+    li::before{
+      content: "+";
+      display: inline-block;
+      position: absolute;
+      left: -25px;
+
+      color: orange;
+      font-size: 20px;
+      font-weight: 700;
+    }
+  }
+  &--photo{
```

Comment:
Może nie widać tego na pierwszy rzut oka, ale przy każdym obrazku jest numer - ot taki wymysł grafika. Można to dodać jako element ::before z właściwością content, bo jest to tylko dekoracja a coś semantycznego (więc nie umieszczajmy tego w htmlu)